### PR TITLE
Minor fixes to various docs

### PR DESCRIPTION
**Pull request by FaithOmbongi**
Fix scopedMembers API: No new APIs; Just fixing APIs that were not surfaced in the docs.
- Surface **delta** API.
- Add missing scopedMembers API v1.0.

Fix query parameter support in User resource.
Fix interchanged icons for advanced query parameters article.

### CODE REVIEW

File: api-reference/v1.0/api/directoryrole-list-scopedadministrators.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "List scopedMembers for a directory role"
```
Comment (Lauragra) [inline api-reference/v1.0/api/directoryrole-list-scopedadministrators.md:2]: The file name here should be directoryrole-list-scopedmembers.

File: api-reference/v1.0/toc.yml [hunk 2]
```diff
@@ -4430,6 +4430,10 @@ items:
             href: api/directoryrole-post-members.md
           - name: Remove member
             href: api/directoryrole-delete-member.md
+          - name: List scoped administrators
```
Comment (Lauragra) [inline api-reference/v1.0/toc.yml:4433]: I think you want the TOC title here to be List scoped members. You'll also want to update the href when you rename the file.

File: api-reference/v1.0/resources/directoryrole.md [hunk 3]
```diff
@@ -27,6 +27,7 @@ This resource supports:
 |[List members](../api/directoryrole-list-members.md) |[directoryObject](directoryobject.md) collection| Get the users that are members of the directory role from the members navigation property.|
 |[Remove a member](../api/directoryrole-delete-member.md) |[directoryObject](directoryobject.md)| Remove a user from the directory role.|
 |[Activate directoryRole](../api/directoryrole-post-directoryroles.md) |[directoryRole](directoryrole.md) | Activate a directory role.|
+|[List scoped-role members](../api/directoryrole-list-members.md) |[scopedRoleMembership](scopedrolemembership.md) collection| List the members of this directory role that are scoped to [administrative units](administrativeunit.md), through the scopedRoleMembership resource collection.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/directoryrole.md:30]: The link text here should be List scopedMembers.

File: api-reference/beta/api/directoryrole-list-scopedadministrators.md [hunk 4]
```diff
@@ -44,15 +44,15 @@ Do not supply a request body for this method.
 If successful, this method returns a `200 OK` response code and collection of [scopedRoleMembership](../resources/scopedrolemembership.md) objects in the response body.
```
Comment (Lauragra) [inline api-reference/beta/api/directoryrole-list-scopedadministrators.md:44]: You'll want to rename this file as well per my comment on the v1.0 topic; the file name should include the name of the navigation property.

File: api-reference/v1.0/api/directoryrole-list-scopedadministrators.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+description: "Retrieve a list of scopedRoleMembership objects for a directory role."
+author: "abhijeetsinha"
+localization_priority: Normal
+ms.prod: "directory-management"
+doc_type: apiPageType
+---
+
+# List scopedMembers for a directory role
```
Comment (Lauragra) [inline api-reference/v1.0/api/directoryrole-list-scopedadministrators.md:10]: Please add an entry to the changelog to call out the new operation in v1.0.

**Review by Lauragra — APPROVED** [review]

**Comment by Lauragra** [comment]
@FaithOmbongi can you take care of the merge conflict? Thanks!

**Comment by CelesteDG** [comment]
PR has been approved and clean. Merging to main.

**Pull request merged**